src/test/ui/print_type_sizes/async.stdout

```diff
@@ -0,0 +1,25 @@
+print-type-size type: `[static generator@$DIR/async.rs:10:32: 13:2]`: 16386 bytes, alignment: 1 bytes
+print-type-size     end padding: 16386 bytes
```

Comment:
Nit: Do you know why it does this? Does this type really only consist of padding? 😕

Reply:
hmm. should we print "padding/unknown" as a special case here, in that case?